presto-hive/src/main/java/com/facebook/presto/hive/cache/HiveCachingHdfsConfiguration.java

```diff
@@ -63,12 +67,8 @@ public Configuration getConfiguration(HdfsContext context, URI uri)
             try {
                 FileSystem fileSystem = (new Path(factoryUri)).getFileSystem(hiveHdfsConfiguration.getConfiguration(context, factoryUri));
                 checkState(fileSystem instanceof ExtendedFileSystem);
-                return new CachingFileSystem(
-                        factoryUri,
-                        factoryConfig,
-                        cacheManager,
-                        (ExtendedFileSystem) fileSystem,
-                        cacheValidationEnabled);
+                return cacheFactory.createCachingFileSystem(factoryConfig, factoryUri,
```

Comment:
one param per line leaving the first line empty

Reply:
done